usr/share/rear/lib/nbkdc-functions.sh

```diff
@@ -0,0 +1,54 @@
+# locate NovaBACKUP DC client configuration, referenced by
+#
+# prep/NBKDC/default/40_prep_nbkdc.sh
+# skel/NBKDC/etc/scripts/system-setup.d/59-start-nbkdc-client.sh
+
+
+# Set the NovaBACKUP DC default directory
+NBKDC_DIR=/opt/NovaStor/DataCenter
+
+
+function get_nbkdc_dir() {
+
+# check for running process rcmd-executor to determine its parent path
+NBKDC_DIR=$(ps -ef | grep rcmd-executor | grep -v grep | awk -F\  '{print $8}' | sed 's/\/rcmd-executor*//g')
+   if  [ ! -d "$NBKDC_DIR" ] ; then
+       echo "Cannot find running NovaBACKUP DataCenter Agent!!"
+       exit 2
+   fi
+}
+
+
+[[ ! -d "$NBKDC_DIR" ]] && get_nbkdc_dir
+
+
+CLIENT_INI=$NBKDC_DIR/conf/client.properties
+[[ -z "$CLIENT_INI" ]] && return
+
+while IFS== read key value ; do
+    case "$key" in
+        hiback_install_dir) NBKDC_HIB_DIR="$value" ;;
```

Comment:
Sorry about that, I was sure my editor had it properly aligned.
I will change it ;-)